pkg/asset/internal/templates.go

```diff
@@ -11,14 +29,42 @@ clusters:
 users:
 - name: kubelet
   user:
-    client-certificate-data: {{ .KubeletCert}}
-    client-key-data: {{ .KubeletKey }}
+    token: {{ .KubeletTLSBootstrapToken }}
 contexts:
 - context:
     cluster: local
     user: kubelet
 `)
 
+var AuthTokenFileTemplate = []byte(`{{ .KubeletTLSBootstrapToken }},system:bootstrapper,,"system:bootstrap"
+`)
+
+var CSRApproverRoleBindingTemplate = []byte(`kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: approve-node-client-csr
+rules:
+- apiGroups: ["certificates.k8s.io"]
+  resources: ["certificatesigningrequests/nodeclient"]
+  verbs: ["create"]
+- apiGroups: ["certificates.k8s.io"]
```

Comment:
note for later:

There's actually a `system:node-bootstrapper` role we can just bind to https://github.com/kubernetes/kubernetes/blob/3a0d8f8fea75140e4a38f5165a75e93bcd28c60c/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/cluster-roles.yaml#L724-L742